src/Admin/Admin.php

```diff
@@ -13,11 +13,11 @@
 
 namespace Sonata\AdminBundle\Admin;
 
-@trigger_error(
-    'The '.__NAMESPACE__.'\Admin class is deprecated since version 3.1 and will be removed in 4.0.'
-    .' Use '.__NAMESPACE__.'\AbstractAdmin instead.',
-    E_USER_DEPRECATED
-);
+@trigger_error(sprintf(
+    'The %s\Admin class is deprecated since version 3.1 and will be removed in 4.0. Use %s\AbstractAdmin instead.',
+    __NAMESPACE__,
+    __NAMESPACE__
```

Comment:
```suggestion
    'The %1$s\Admin class is deprecated since version 3.1 and will be removed in 4.0. Use %1$s\AbstractAdmin instead.',
    __NAMESPACE__
```

Reply:
Yes, we can do that. Although i prefer to avoid the numbering of placeholders.

Reply:
This is not a blocker on my side either.